src/swap/components/SwapSettingsSlippageTitle.tsx

```diff
@@ -0,0 +1,18 @@
+import { cn } from '../../styles/theme';
+import type { SwapSettingsSlippageTitleReact } from '../types';
+
+export function SwapSettingsSlippageTitle({
+  children,
+  className,
+}: SwapSettingsSlippageTitleReact) {
+  return (
+    <h3
+      className={cn(
+        'mb-2 font-semibold text-base text-gray-950 leading-normal dark:text-gray-50',
```

Comment:
`text-gray-950`

wait a second, these all colors will break when custom theming is implemented.

Reply:
Won't custom theming just override them?

Reply:
Where do you see it breaking? Dark theme seems to be working on playground and in /site